infra/docker/ci/Dockerfile

```diff
@@ -23,13 +21,25 @@ RUN apt-get install -y build-essential curl python${PYTHON_VERSION} \
 
 
 # Install Go
+ENV GOLANG_VERSION 1.14.1
+
 RUN curl -O https://storage.googleapis.com/golang/go${GOLANG_VERSION}.linux-amd64.tar.gz && \
     tar -xvf go${GOLANG_VERSION}.linux-amd64.tar.gz && chown -R root:root ./go && mv go /usr/local
+
 ENV GOPATH /go
 ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
-
-# Add contents of local Feast repository to image (execute from Feast root)
-COPY . /feast/
-
-# Install all dependencies
-RUN cd /feast && make install-ci-dependencies
\ No newline at end of file
+ENV PATH="$HOME/bin:${PATH}"
+
+# Install Protoc and Plugins
+ENV PROTOC_VERSION 3.11.4
```

Comment:
Should we use 3.11.4 or 3.10.0?

Reply:
Yea, 3.10 would be better. Let me see if that works.